src/instructlab/eval/mtbench.py

```diff
@@ -9,48 +12,85 @@ class MT_Bench_Evaluator(Evaluator):
     Child class of an Evaluator for Multi-turn Benchmark (MT-Bench)
 
     Attributes
-        server_url  vLLM server endpoint
+        model_name  Name of the model to evaluate
     """
 
-    def __init__(self, model_path, server_url: str) -> None:
-        super().__init__(model_path)
-        self.server_url = server_url
+    def gen_answers(self, server_url) -> None:
+        """
+        Asks questions to model
+
+        Attributes
+            server_url  Model server endpoint
+        """
+        gen_api_answer.run(self.model_name, openai_api_base=server_url)
 
-    def run(self) -> tuple:
+    def judge_answers(self, judge_model_name, server_url) -> tuple:
         """
-        Runs MT-Bench evaluation
+        Runs MT-Bench judgment
+
+        Attributes
+            judge_model_name  Name of the judge model
+            server_url  Model server endpoint
 
         Returns:
             overall_score   MT-Bench score for the overall model evaluation
             qa_pairs        Question and answer pairs from the evaluation
         """
-        overall_score: float = 0.0
-        qa_pairs: list[tuple] = []
-        return overall_score, qa_pairs
+        return gen_judgment.run(
+            self.model_name,
+            judge_model_name=judge_model_name,
+            parallel=40,
```

Comment:
Did we decide if we want to set this value somewhere or just hardcode 40 somehow? That's a smaller detail and shouldn't hold up the merging of this PR.

Reply:
Good question.  Maybe we can do some autotuning.

Reply:
I went ahead an pulled the variable out to the top as an option (max_workers)